scripts/ntx_data/utils/ntx_add_transplant_id_to_labs.py

```
@@ -0,0 +1,147 @@
+from typing import Dict, List
+from scripts.ntx_data.utils import processed_data_dir
```

Pylint (4.0.2)

[error] No name 'processed_data_dir' in module 'scripts.ntx_data.utils'

(E0611)